DFKPS/iocBoot/iocDFKPS-IOC-01/config.xml

```diff
@@ -3,8 +3,8 @@
 <config_part>
 <ioc_desc>Danfysik power supply</ioc_desc>
 <macros>
-    <macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM Port" hasDefault="NO" />
-    <macro name="BAUD" pattern="^[0-9]+$" description="Serial communication baud rate, defaults to 9600 for most Danfysiks, 115200 for the 9X00 versions" defaultValue="9600" hasDefault="YES" />
+    <xi:include href="../../../COMMON/PORT.xml" />
+    <xi:include href="../../../COMMON/BAUD9600.xml" />
 	<macro name="POLARITY" pattern="^(BIPOLAR|UNIPOLAR)$" description="Polarity type, BIPOLAR - polarity is stable; defaults to BIPOLAR" defaultValue="BIPOLAR" hasDefault="YES" />
```

Comment:
I don't think we can use the common files as the default value depends on danfysik model (9600 or 115200)